include/git2/hook.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) the libgit2 contributors. All rights reserved.
+ *
+ * This file is part of libgit2, distributed under the GNU GPL v2 with
+ * a Linking Exception. For full terms see the included COPYING file.
+ */
+#ifndef INCLUDE_git_hook_h__
+#define INCLUDE_git_hook_h__
+
+#include "common.h"
+#include "types.h"
+
+/**
+ * @file git2/hook.h
+ * @brief Git hook management routines
+ * @defgroup git_hook Git hook management routines
+ * @ingroup Git
+ * @{
+ */
+GIT_BEGIN_DECL
+
+typedef int (*git_hook_foreach_cb)(const char *hook_name, void *payload);
+
+GIT_EXTERN(int) git_hook_enumerate(
+	git_repository *repo,
+	git_hook_foreach_cb callback,
+	void *payload
+);
```

Comment:
Our coding style has the closing brace on the same line as the last argument. Also applies for the following functions.